client/src/pages/Home.js

```diff
@@ -58,9 +62,31 @@ function Home({ onPaste }) {
         value={expireTime}
         onChange={event => setExpireTime(parseInt(event.target.value))}
       ></SelectTime>
+
+      <EncryptCheckbox
+        value={isEncrypted}
+        label={'Encrypt my Pasta'}
+        onChange={() => setIsEncrypted(!isEncrypted)}
+      />
+
+      <PasswordInput
+        value={password}
+        onChange={event => setPassword(event.target.value)}
+      />
+
       <SubmitButton
-        onClick={() => doPost({ value: pasteValue, author, expireTime })}
-        disabled={!pasteValue || !author || loading}
+        onClick={() =>
+          doPost({
+            value: pasteValue,
+            author,
+            expireTime,
+            password,
+            isEncrypted: isEncrypted
```

Comment:
You add a state as a property here `isEncrypted` to the doPost function coming from the `usePostPaste` hook. However I don't see you adding/drilling it through inside the function itself... what is the purpose of this state here? 
Maybe you wanted to set the state?

Reply:
```

Reply:
That will be packed in a request. This determines whether pasta should be encrypted or not.

![image](https://user-images.githubusercontent.com/60475379/78010139-50de4080-7342-11ea-9915-da9f1ac05fb2.png)

Reply:
```suggestion
            isEncrypted
```